sw/device/silicon_creator/lib/cert/dice_cwt.c

```diff
@@ -0,0 +1,165 @@
+            0xf295b128,
+            0x50f49882,
+            0xbbdefa29,
+        },
+    .version = 0,
+};
+
+const sc_keymgr_ecc_key_t kDiceKeyUds = {
+    .type = kScKeymgrKeyTypeAttestation,
+    .keygen_seed_idx = kFlashInfoFieldUdsKeySeedIdx,
+    .keymgr_diversifier = &kUdsKeymgrDiversifier,
+    .required_keymgr_state = kScKeymgrStateCreatorRootKey,
+};
+const sc_keymgr_ecc_key_t kDiceKeyCdi0 = {
+    .type = kScKeymgrKeyTypeAttestation,
+    .keygen_seed_idx = kFlashInfoFieldCdi0KeySeedIdx,
+    .keymgr_diversifier = &kCdi0KeymgrDiversifier,
+    .required_keymgr_state = kScKeymgrStateOwnerIntermediateKey,
+};
+const sc_keymgr_ecc_key_t kDiceKeyCdi1 = {
+    .type = kScKeymgrKeyTypeAttestation,
+    .keygen_seed_idx = kFlashInfoFieldCdi1KeySeedIdx,
+    .keymgr_diversifier = &kCdi1KeymgrDiversifier,
+    .required_keymgr_state = kScKeymgrStateOwnerKey,
+};
+
+// PubKeyECDSA256 = {               ; COSE_Key [RFC9052 s7]
+//     1 : 2,                       ; Key type : EC2
+//     3 : AlgorithmES256,          ; Algorithm : ECDSA w/ SHA-256
+//     -1 : 1,                      ; Curve: P256
+//     -2 : bstr,                   ; X coordinate, big-endian
+//     -3 : bstr                    ; Y coordinate, big-endian
+// }
+rom_error_t dice_uds_tbs_cert_build(cert_key_id_pair_t *key_ids,
+                                    ecdsa_p256_public_key_t *uds_pubkey,
+                                    uint8_t *tbs_cert, size_t *tbs_cert_size) {
+  struct CborOut kCborOutHandle;
+
+  struct CborOut *pCborOut = &kCborOutHandle;
+
+  HARDENED_RETURN_IF_ERROR(
+      cbor_write_out_init(pCborOut, tbs_cert, *tbs_cert_size));
+  HARDENED_RETURN_IF_ERROR(cbor_map_init(pCborOut, 5));
+  HARDENED_RETURN_IF_ERROR(
+      cbor_write_pair_uint_uint(pCborOut, kCoseKeyKtyLabel, kCoseKeyKtyEc2));
+  HARDENED_RETURN_IF_ERROR(cbor_write_pair_uint_int(pCborOut, kCoseKeyAlgLabel,
+                                                    kCoseKeyAlgEcdsa256));
+  HARDENED_RETURN_IF_ERROR(
+      cbor_write_pair_int_uint(pCborOut, kCoseEc2CrvLabel, kCoseEc2CrvP256));
+  HARDENED_RETURN_IF_ERROR(cbor_write_pair_int_bytes(pCborOut, kCoseEc2XLabel,
+                                                     (uint8_t *)uds_pubkey->x,
+                                                     sizeof(uds_pubkey->x)));
+  HARDENED_RETURN_IF_ERROR(cbor_write_pair_int_bytes(pCborOut, kCoseEc2YLabel,
+                                                     (uint8_t *)uds_pubkey->y,
+                                                     sizeof(uds_pubkey->y)));
+  *tbs_cert_size = CborOutSize(pCborOut);
+
+  return kErrorOk;
+}
+
+rom_error_t dice_cdi_0_cert_build(hmac_digest_t *rom_ext_measurement,
+                                  uint32_t rom_ext_security_version,
+                                  cert_key_id_pair_t *key_ids,
+                                  ecdsa_p256_public_key_t *cdi_0_pubkey,
+                                  uint8_t *cert, size_t *cert_size) {
```

Comment:
Can you add `// TODO(lowRISC/opentitan:#24281): implement body` comments in these functions that are not yet implemented? (https://opentitan.org/book/doc/contributing/detailed_contribution_guide/index.html?highlight=todo#what-about-todo-comments)